libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -68,12 +71,13 @@
 NSString * const kSyncManagerResponseTotalSize = @"totalSize";
 NSString * const kSyncManagerResponseNextRecordsUrl = @"nextRecordsUrl";
 NSString * const kSyncManagerRecentItems = @"recentItems";
+NSString * const kSyncManagerLastModifiedDate = @"LastModifiedDate";
 
 // dispatch queue
 char * const kSyncManagerQueue = "com.salesforce.smartsync.manager.syncmanager.QUEUE";
 
 // block type
-typedef void (^SyncUpdateBlock) (NSString* status, NSInteger progress, NSInteger totalSize);
+typedef void (^SyncUpdateBlock) (NSString* status, NSInteger progress, NSInteger totalSize, long long maxTimeStamp);
```

Comment:
Okay.  I'd prefer we were doing those conversions on the tier boundaries, since `NSTimeInterval` is the standard way that time intervals are utilized in Objective-C.  But I don't feel strongly about it.